ocs_ci/ocs/ui/base_ui.py

```diff
@@ -382,6 +382,7 @@ def login_ui():
         if headless:
             chrome_options.add_argument("--headless")
 
+        chrome_options.add_argument("window-size=1920,1500")
```

Comment:
does this work? driver.maximize_window() ?

Reply:
@vasukulkarni 
screenshot with this option [chrome_options.add_argument("window-size=1920,1500")]
![image](https://user-images.githubusercontent.com/61982127/117066734-c4b2ff80-ad31-11eb-8d0d-640b5cd76ede.png)

screenshot without this option 
![image](https://user-images.githubusercontent.com/61982127/117067063-37bc7600-ad32-11eb-92c3-361568f0fcaf.png)